cms.php

```diff
@@ -1,72 +1,69 @@
+    PDO::FETCH_ASSOC);
+
+$query= $db->prepare("SELECT `email`, `phone`, `about`,`deleted`
+FROM `cms` WHERE `deleted` = 0 ORDER BY `id` DESC LIMIT 1;");
+
+$query1= $db->prepare("SELECT `project_id`, `project_img`, 
+`project_title`,`project_text`, `project_link` FROM `projects` WHERE 
+`project_delete` = 0 ORDER BY `project_id` ASC;");
+
+$query->execute();
+$query1->execute();
 
+$result = $query->fetchAll();
+$result1 = $query1->fetchAll();
+
+$email = $result [0]['email'];
+$phone = $result [0]['phone'];
+$about = $result [0]['about'];
+$del = $result [0]['deleted'];
+
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="cms.css">
+    <title>Portfolio Input Page</title>
+</head>
 <h3>Welcome to the Portfolio Input Page</h3>
 
-<form method="post" action="#">
-    <input type="text" name="email">email
+<form method="post" action="cms.php">
+    <input type="text" value="<?php echo $email; ?>" name="email">email
     <input type="submit" value="update">
 </form><br>
-<form method="post" action="#">
-    <input type="tel" name="tel">Telephone Number
+<form method="post" action="cms.php">
+    <input type="tel" value="<?php echo $phone; ?>" name="tel">Telephone Number
     <input type="submit" value="update">
 </form><br>
-<form method="post" action="#">
-    <input type="text" name="about">About me paragraph
+<form method="post" action="cms.php">
+    <textarea rows="8" cols="150"  name="about" form="about">
+        <?php echo $about; ?></textarea>
     <input type="submit" value="update">
 </form><br>
 
-<form method="post" action="#">
-    <h4>Project holder 1</h4><br>
-    <input type="text" name="project-img">Image file name<br>
-    <input type="text" name="project-title">Project Title (max 100 characters)<br>
-    <input type="text" name="project-text">Project Description (max 2000 characters)<br>
-    <input type="text" name="project-link">Project link URL<br>
-    <input type="submit" value="update">
-</form>
-
-<form method="post" action="#">
-    <h4>Project holder 2</h4><br>
-    <input type="text" name="project-img">Image file name<br>
-    <input type="text" name="project-title">Project Title (max 100 characters)<br>
-    <input type="text" name="project-text">Project Description (max 2000 characters)<br>
-    <input type="text" name="project-link">Project link URL<br>
-    <input type="submit" value="update">
-</form>
+    <h4>Project holder</h4>
+<?php
 
-<form method="post" action="#">
-    <h4>Project holder 3</h4><br>
-    <input type="text" name="project-img">Image file name<br>
-    <input type="text" name="project-title">Project Title (max 100 characters)<br>
-    <input type="text" name="project-text">Project Description (max 2000 characters)<br>
-    <input type="text" name="project-link">Project link URL<br>
-    <input type="submit" value="update">
-</form>
+foreach ($result1 as $list) {
```

Comment:
Maybe turn this into a function, also look into embedding php in html as it might be easier.

Reply:
Yes this is a good idea. I will add this change to the next branch